project/tests/com/google/daggerquery/plugin/GraphConverterTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.google.daggerquery.plugin;
+
+import com.google.common.graph.MutableNetwork;
+import com.google.common.graph.Network;
+import com.google.common.graph.NetworkBuilder;
+import com.google.daggerquery.protobuf.autogen.BindingGraphProto;
+import com.google.daggerquery.protobuf.autogen.BindingGraphProto.BindingGraph.ListWithDependencies;
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.Map;
+
+public class GraphConverterTest {
```

Comment:
Do you think it might be worth putting a couple more test cases here, such as what happens when the network doesn't contain anything? Or calling the methods with some other bad input. Maybe that couldn't happen, I'm not sure?

Reply:
Yeah, since method `makeAdjacencyList` is public it makes sense to show what will happen in worse situation!

I've added tests with passing null arguments, with passing empty network (it's also incorrect because we assume that in binding graph we have at least one node - main component) and with passing invalid source node.